lib/drivers/medtronic/medtronicDriver.js

```diff
@@ -161,7 +161,13 @@ module.exports = function (config) {
       }
       ctr += struct.pack(bytes, ctr, '7bi', 0, 0, 0, partsPerPage, 0x10, 0x10, 0x00, payload.length+secondPacketLength);
     } else {
-      ctr += struct.pack(bytes, ctr, '7bi', 0, 0, 0, 0, 0, 0, 0, payload.length);
```

Comment:
This could be changed using new insight into the meanings of these bytes to make the code easier to understand (especially from a sense of purpose), but it's not critical to it working.